prettier.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  bracketSpacing: false,
```

Comment:
I know the point here isn't to paint the bikeshed, but I personally like bracket spacing *ducks*

Reply:
If you look at the delta, this is ported from the `.prettierrc` that you added.

Is there rationalization for the value you used?

Reply:
I had just copied over what's used in all of the other libraries 🤷‍♂️   I doubt those choices were really scrutinized.  I don't have a strong opinion here, just a preference for defaults when nobody otherwise cares. What do you think?

Reply:
I've updated the config to a minimal set (IMO). PTAL.